module2/homework/grayscaleImages/test.cpp

```diff
@@ -0,0 +1,190 @@
+#include <algorithm>
+#include <array>
+#include <utility>  // for std::pair<>
+#include <vector>
```

Comment:
Yuo could remove array and vector here since they are in the header you have included

Reply:
+1